internal/cmd/deploy.go

```diff
@@ -69,6 +69,7 @@ microservices: []
 
 	// Register flags
 	cmd.Flags().StringVarP(&opt.Filename, "file", "f", "", "YAML file containing resource definitions for Controllers, Agents, and Microservice to deploy")
+	cmd.Flags().BoolVarP(&opt.Local, "local", "l", false, "Local deployment using docker containers. All hosts will be replaced by 0.0.0.0")
```

Comment:
Do you think we should infer local by the other args they specify (like localhost for --host)? I know the other commands have --local but wondering if you think it would be a good idea to not required the flag.

Reply:
We could infer from the host and see how that goes. We could end up with hybrid solutions where the agent is a local docker image and the controller is remote. Or vice versa.